README.md

```diff
@@ -40,6 +40,12 @@ In order to build Mozart 2, you need the following tools on your computer:
 *   Tcl/Tk 8.5 or 8.6 (with development files)
 *   emacs
 
+For building CLANG/LLVM:
+*   LibXML2-dev (tested with version 2.9.3)
+*   OCaml-findlib
+*   libCTypes-OCaml-dev (>= 0.4 - available in Debian Unstable as of Jan. 2016)
```

Comment:
Does LLVM/Clang really require this ?
According to the docs, these are not required: http://llvm.org/docs/GettingStarted.html#software